content/presseberichte/bnn_2016_09_05.md

```diff
@@ -1,5 +1,5 @@
 ---
-date: "2015-09-05T14:11:29+02:00"
+date: "2016-09-05T14:11:29+02:00"
 draft: false
 title: "Genuss aus der Vogelperspektive"
 slug: "bnn_artikel"
```

Comment:
Mir fiel hier grade auf, die Slugs müssen ja eindeutig sein und wir haben nicht nach Jahreszahlen in Ordner katalogisiert - daher hier vllt. noch das Datum aufnehmen (bnn-artikel-2016-09-05 oder bnn-artikel-2016-vogelperspektive).